pkg/parser/pipelineyml/visitor_ref_op.go

```diff
@@ -322,16 +320,6 @@ func (v *RefOpVisitor) handleOneRefOpOutput(refOp RefOp) (replaced string) {
 		}
 	}
 
-	actions := v.allActions
-	if actions != nil {
-		action, ok := actions[ActionAlias(refOp.Ref)]
-		// 假如 action 是 snippet 类型，走下面的校验
-		if ok && IsSnippetType(action.Type) {
-			v.handleSnippetOneRefOutput(refOp, action)
-			return
-		}
```

Comment:
Is that mean no more SnippetType of action?

Reply:
yes. treat snippet as normal action.